### DISCUSSION TITLE
scrum prep Milestone 2 - 1

### DISCUSSION THREAD

**Discussion by MrL9000** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#98] [General]
Write your scrum here.

**Comment by MrL9000** [comment]
I've mostly spent this week looking at the code to start getting a feel for the code and creating tasks. Outside of speaking with the managers to fully get the scores down, and getting the hang of Angular, I'm set to do the tasks I've self assigned.

**Comment by sebasagir** [comment]
I was not able to do much this week aside from attending the team meetings and getting familiar with the milestone 2 objectives due to having a complicated week

**Comment by warm007** [comment]
This week, I attended the team meeting where we discussed our plan for milestone #2 and divided the tasks accordingly. While I haven't had the chance to implement anything concrete yet, I've been reviewing Angular and the source code. My goal is to have something visible by the next team meeting.

**Comment by AngelMartinez33** [comment]
helped with the milestone 1 presentation and organized a meeting for the metrics team to start working on the objectives for milestone 2.

**Comment by AlexisTraversoPR** [comment]
Had a 1 on 1 call with one of the managers where we gained a better understanding for how the notification system works. We managed to order the notifications correctly and came up with my task for the next week

**Comment by Brandell13** [comment]
This week I was studying the code and meeting in the two meetings (general meeting and team meeting). Soon I will be creating the task for this milestone.

**Comment by Yahid1** [comment]
This week I clarified milestone objectives 2. Almost finished the documentation on how to run an isolated backend. Made tasks available for the individuals in both teams. Explained coding segments for some of the developers. Organized the project Board to include tasks corresponding to the milestone 2.

**Comment by Janiel777** [comment]
No pude hacer mucho esta semana. A sido una semana complicada pero aún así, asistí a las reuniones y me e estado manteniendo informado al respecto.

**Comment by JaseyAcevedo** [comment]
Dealing with a hell week but Ive seen a posible opportunity to add some dummy notificación with their triggers for some future important events like low gas or tires and just give the hardware control of those triggers

**Comment by Alej24** [comment]
Attended two meetings this week and have two tasks assigned which I will hopefully be able to work on soon.

**Comment by IvanierB** [comment]
This week we started with milestone 2, so I got familiarized with the objectives and self assigned 1 documentation task and 1 coding task to start working on.

**Comment by dAninAd1248** [comment]
I was quite busy this week so I only thought about possible tasks.

**Comment by juan-ortiz92** [comment]
I have been revising the notes given by the professor regarding the first milestone and further worked on notifications implementation.

**Comment by diegorobles2** [comment]
Attended the meetings and later today will create the tasks for spring #1

**Comment by AleDiazz** [comment]
finished studying code to start coding task for front end, attended meetings and will be working on making my first task today

**Comment by rafa-quiles** [comment]
Attended meeting, not much else due to other classes, but will make a task for myself today

**Comment by marcosarraiza** [comment]
Mostly spent this week just getting familiarized with the previous code and getting my tasks ready for the sprint.

**Comment by JoelTorres20** [comment]
Assigned myself a task to work on this weekend, but not much else this week.

**Comment by yamayrie** [comment]
This week I attended a general meeting and continued getting familiarized with the project to be able to create tasks

**Comment by dylanDeJesus21** [comment]
Studied RumBus code
